### PR TITLE
Add type definitions for pretty-data

### PR DESCRIPTION
**Pull request by ziishaned**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/pretty-data/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Type definitions for pretty-data 0.40
+// Project: https://github.com/sidorares/json-bigint#readme
```
Comment (demensky) [inline types/pretty-data/index.d.ts:2]: Did you mean [this link](https://github.com/vkiryukhin/pretty-data)?

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## pretty-data ([<kbd>unpkg</kbd>](https://unpkg.com/browse/pretty-data@latest/))
was missing the following properties:
1. pd
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 82ccbf6acf1f8dc88491fd875164f94ee502ebba
</p>

**Comment by ziishaned** [comment]
@demensky can you please review again

**Comment by demensky** [comment]
Following from [the source code](https://github.com/vkiryukhin/pretty-data/blob/22a34a534745714ea974f72db2e9bf7561b3fdc7/pretty-data.js#L331) of the library, [it exports one field named `pd`](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/49083#issuecomment-716143030). An error occurs when you try to access methods directly.

example1

You should probably use this:
 
```ts
export const pd: {
  // …
```

**Comment by demensky** [comment]
@ziishaned don't forget to update your tests.

File: types/pretty-data/pretty-data-tests.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import pd = require('pretty-data');
```
Comment (demensky) [inline types/pretty-data/pretty-data-tests.ts:1]:
```suggestion
import {pd} from 'pretty-data';
```

**Review by demensky — CHANGES_REQUESTED** [review]

**Review by demensky — APPROVED** [review]

**Comment by ziishaned** [comment]
Ready to merge

**Pull request merged**